main/init/sbt/src/mill/main/sbt/SbtBuildGenMain.scala

```diff
@@ -123,7 +123,7 @@ object SbtBuildGenMain
     } catch {
       case e: os.SubprocessException => throw RuntimeException(
           "The sbt command to run the `millInitExportBuild` sbt task has failed, " +
-            s"please update the project's sbt version to the latest or our tested version v${sys.props("TEST_SBT_VERSION")}, " +
+            s"please update the project's sbt version to the latest or our tested version v${sys.props.getOrElse("TEST_SBT_VERSION", ???)}, " +
```

Comment:
It's not a good idea to throw an exception while we're about to construct another exception. Also, this is not in the context of the test project, so the `TEST_SBT_VERSION` variable is undefined. 

If we want to hint the sbt version used at Mill build time, we need some generated constant, e.g. by using the `BuildInfo` module. Here is an usage example:

https://github.com/com-lihaoyi/mill/blob/59764e88a2168db8eeb61ecc0f10a171f4c160e4/scalalib/src/mill/scalalib/ZincWorkerModule.scala#L133

It's defined here:

https://github.com/com-lihaoyi/mill/blob/59764e88a2168db8eeb61ecc0f10a171f4c160e4/scalalib/package.mill#L45

Reply:
Sorry I didn't realize this. Fixed.